mandatory/1-fix-functions.js

```diff
@@ -9,42 +9,40 @@
   NOTE:Only make edits inside the function
 
 */
-
-function mood() {
-  let isHappy = true;
-
-  if (isHappy) {
+let isHappy = true;
+ 
+function mood(isHappy){
+ if (isHappy) {
     return "I am happy";
   } else {
     return "I am not happy";
   }
 }
 
 function greaterThan10(num) {
-  let isBigEnough;
 
-  if (isBigEnough) {
+  if (num > 10 ) {
```

Comment:
I would have personally done

let isBigEnough = num > 10

What you've done is correct though